common/interfaces/user_interface/ui_skip_instruction.c

```diff
@@ -74,27 +74,25 @@ void card_detect_scr_init(const char *text) {
   if (data != NULL) {
     data->text = (char *)text;
   }
-  card_detect =
-      lv_task_create(nfc_card_presence_detect, 100, LV_TASK_PRIO_LOW, NULL);
-  card_detect_scr_create();
+  skip_instruction_scr_create();
```

Comment:
in regards to https://github.com/Cypherock/x1_wallet_firmware/pull/217, it was needed that every screen clears previously rendered screen and destructor is called via `LV_EVENT_DELETE`.

best to follow the same pattern decided.

References: 
- https://github.com/Cypherock/x1_wallet_firmware/pull/217/files#diff-9374862707c912d739837b27c0961c631142ddd16643f0d8d050d8de0abc1d89R98,
- https://github.com/Cypherock/x1_wallet_firmware/pull/217/files#diff-9374862707c912d739837b27c0961c631142ddd16643f0d8d050d8de0abc1d89R176 &
- https://github.com/Cypherock/x1_wallet_firmware/pull/217/files#diff-9374862707c912d739837b27c0961c631142ddd16643f0d8d050d8de0abc1d89R182